src/main/steps/sibling/name/post.ts

```diff
@@ -0,0 +1,53 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { getNextStepUrl } from '../..';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { Form } from '../../../app/form/Form';
+import { ValidationError } from '../../../app/form/validation';
+
+@autobind
+export default class SiblingPostController extends PostController<AnyObject> {
+  public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+    const fields = typeof this.fields === 'function' ? this.fields(req.session.userCase) : this.fields;
+    const form = new Form(fields);
+    const { saveAndSignOut, saveBeforeSessionTimeout, _csrf, ...formData } = form.getParsedBody(req.body);
+
+    req.session.errors = form.getErrors(formData);
+
+    const siblingNames = req.session.userCase.siblings?.find(
```

Comment:
variable name could be
```suggestion
    const sibling = req.session.userCase.siblings?.find(
```